package.json

```diff
@@ -65,11 +65,12 @@
     "highcharts": "9.1.2",
     "highcharts-vue": "1.4.0",
     "idna-uts46": "1.1.0",
+    "lodash": "4.17.21",
```

Comment:
web3 actually comes with underscore. you can remove this and use something similar from them instead.